templates/ansible/roles/nodejs/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

It is required to install this way on Debian systems.

I'm happy to contribute, I really like your project.
Pick and choose whatever makes sense and if you need an extra hand
working on something feel free to reach out.

I'm digging into the ansible docs as we speak to be able to write and
better build the playbooks so in a few days I'll have much more to offer.

Digging around I found this project that looks most promising:
http://debops.org/

My aim is to leverage it and your project to make the plug and play
setup and get the aws integration added. I already have an EC2
instance with debian talking with the basic ansible inventory lists
and ad-hoc commands.

The end goal setup is EC2 instance built with nginx, puma, redis,
sidekiq, linked into a postgresql RDS, and hooked into an Elasti-cache
redis all residing in the same VPC.

I'll keep you posted on my progress if that will help.


On 3/13/15 11:08 PM, Andrey Chernih wrote:
>> @@ -0,0 +1,48 @@ +---
> 
> @RDrop what are the benefits of installing latest version of
> nodejs manually? It's only needed as JS runtime for assets pipeline
> and I think version which is installed with apt-get works well.
> 
> --- Reply to this email directly or view it on GitHub: 
> https://github.com/andreychernih/railsbox/pull/14/files#r26435123
> 

- -- 


"This Email is covered by the Electronic Communications Privacy Act,
18 U.S.C. §§ 2510-2521 and is legally privileged. This electronic mail
message contains confidential information and is intended only for the
use of the recipient named above. If the reader of this email is not
the intended recipient, we request immediate notification by return
email so that we can correct the error and request that you destroy
all copies of this misrouted email. If you receive this email in
error, you are hereby notified that any copying, distribution or the
taking of any action in reliance of the contents of this electronic
message is strictly prohibited."
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v1

iQIcBAEBAgAGBQJVA7rFAAoJEDrEH5SwiMkQj3wP/ReLF2aVjaFRLWtKvyYoK/Hg
kbv9IYiPJjRkd5W1Fn6aIAYcPcAUu85GjQS2THZErD7a2ehHZn8s6py08RiBbOX8
XeNaeeubyQ5Mx9uymbUXkd9Ug7skbpBbjqMiK8vHZQMJWJQF9ui0PTbPM3Ggo7pL
vQWHZW3m55wQYuHHGQFVpduK0e7bXVMKaeTKqMdWyOAjBPP1fTFmkH+btejIi9Dt
PLD2qqhtXmhvqcTRZ2v/egqsNYf6jUICD53a3+UwgTP+LFx2VIWuRujsEsNEZgkX
RGa8gVdya/45g4p36opewAc3+KfO5ANHcPrCacVH1S7JyJbkioUEA+Q/nMrg1wDr
Iysisu8pkPC/gAKf5f6Zw0rBDyLAErz0cXd09B6KpnJ3Vo6zRd3RD4QBnIzlZxnn
QSf/GDQgpGVN3l1gQA77azLochUfmssxPhmJal3sLBldPISiHXf4KVbJoeXOohCK
tDnfsDV5e82KHFKPNzjXeCmX/xJUsd5zk9ilMUpLFoMQ9/4IBKFUjjRzeg2rIQfw
21IbJ4IhdmQ3zPqITZ+sdHSEIVehq1OrVPXondp7HaZvBVCK0tb17KVNHkzZfCQI
BaP8OTZkEtGKO9QNDYJduqv2xJrKuhJn1eKDXFGZzkbhsNCSSxYB1s7v/Gp6hd/I
jhn68RudpkVPYEDtmxfW
=xN09
-----END PGP SIGNATURE-----